project/src/pages/main/main.tsx

```diff
@@ -0,0 +1,23 @@
+import {FilmType} from '../../types/film';
+import FilmPromoCard from '../../components/film-promo-card/film-promo-card';
+import PageContent from '../../components/page-content/page-content';
+
+type MainProps = {
+  filmPromoData: FilmType,
+  filmsData: FilmType[]
+}
+
+function Main({filmPromoData, filmsData}: MainProps) {
+  return (
+    <>
+      <FilmPromoCard
+        filmPromoData={filmPromoData}
+      />
```

Comment:
В столбик располагают обычно после 3 пропсов и больше
```suggestion
      <FilmPromoCardfilmPromoData={filmPromoData} />
```